pkg/controller/servicebindingrequest/servicebinder.go

```diff
@@ -247,7 +235,15 @@ func (b *serviceBinder) bind() (reconcile.Result, error) {
 	sbrStatus := b.sbr.Status.DeepCopy()
 
 	b.logger.Info("Saving data on intermediary secret...")
-	secretObj, err := b.secret.commit(b.envVars)
+
+	secretOwnerReference := v1.OwnerReference{
+		Name:       b.sbr.Name,
+		UID:        b.sbr.UID,
+		Kind:       b.sbr.Kind,
+		APIVersion: b.sbr.APIVersion,
+	}
+
+	secretObj, err := b.secret.commit(b.envVars, secretOwnerReference)
```

Comment:
we can use here `b.secret.createOrUpdate` instead, and remove `commit` function as duplicate.

Reply:
removed 3a5341c